### DISCUSSION TITLE
Add ESlint always_specify_type rules

### DISCUSSION THREAD

**Discussion by PhilipPurwoko** [eslint/eslint#15067] [Ideas] [answered]
Hello, I am currently working on Typescript project with my team. Also, I was familiar with Flutter. The good things I love in Flutter analysis is that you can enable the option for `always_specify_type`. Because Typescript was superset of Javascript, what's the point if I don't use type in my entire code. I think it would be very good to have that equivalent rules. Thank you

**Comment by mdjermanovic** [comment, marked as answer]
Hi @PhilipPurwoko,

Typescript rules and everything related to typescript-eslint is maintained in https://github.com/typescript-eslint/typescript-eslint repository.

You can check the list of rules provided by [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin), maybe there is already a rule you are lookin for.
